src/blocktree.rs

```diff
@@ -399,6 +399,18 @@ impl Blocktree {
         self.meta_cf.get(&MetaCf::key(slot_height))
     }
 
+    pub fn clear_slot(&self, slot_height: u64) -> Result<()> {
```

Comment:
@carllin - this seems to be enough to clear out a slot, do you think more is needed though?

Reply:
Hmm, what about the actual coding/data blobs, should those get deleted as well? Otherwise, we won't have any idea they exist